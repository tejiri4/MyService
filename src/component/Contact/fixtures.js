export const inputTypes = [
  {
    type: 'text',
    placeholder: 'Full name',
    key: 'fullname'
  },
  {
    type: 'email',
    placeholder: 'Email address',
    key: 'email'
  },
  {
    type: 'text',
    placeholder: 'Subject',
    key: 'subject'
  },
]

export const contactDetails = [
  {
    icon: 'https://res.cloudinary.com/store-manager/image/upload/v1563274461/facebook-placeholder-for-locate-places-on-maps.svg',
    alt: 'location',
    header: 'Location',
    body: '234 Ikorodu Road, Lagos, Nigeria.',
    showBorder: true
  },
  {
    icon: 'https://res.cloudinary.com/store-manager/image/upload/v1563274461/24-hour-support.svg',
    alt: 'phone-number',
    header: 'Phone number',
    body: '08114320486',
    showBorder: true
  },
  {
    icon: 'https://res.cloudinary.com/store-manager/image/upload/v1563274461/mail-black-envelope-symbol.svg',
    alt: 'email',
    header: 'Email address',
    body: 'tejirimatthew4@gmail.com'
  },
];