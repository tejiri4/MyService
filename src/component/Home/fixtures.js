export const socialMediaInfos = [
  {
    key: 'twitter',
    src: 'https://res.cloudinary.com/store-manager/image/upload/v1560940981/twitter.svg',
    alt: 'twitter-icon'
  },
  {
    key: 'instagram',
    src: 'https://res.cloudinary.com/store-manager/image/upload/v1560941286/instalgram.svg',
    alt: 'instalgram-icon'
  },
  {
    key: 'facebook',
    src: 'https://res.cloudinary.com/store-manager/image/upload/v1560941284/facebook.svg',
    alt: 'facebook-icon'
  }
]
