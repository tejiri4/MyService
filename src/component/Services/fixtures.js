export const ServiceCardDetails = [
  {
    img: 'https://res.cloudinary.com/store-manager/image/upload/v1561031570/photo-camera.svg',
    title: 'Photography',
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.'
  },
  {
    img: 'https://res.cloudinary.com/store-manager/image/upload/v1561031570/website-design-symbol.svg',
    title: 'Web design',
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent eget iaculis augue.'
  },
  {
    img: 'https://res.cloudinary.com/store-manager/image/upload/v1561031571/programming.svg',
    title: 'Web application',
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent eget iaculis augue.'
  },
  {
    img: 'https://res.cloudinary.com/store-manager/image/upload/v1561031546/mobile.svg',
    title: 'Mobile application',
    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.'
  }
]
