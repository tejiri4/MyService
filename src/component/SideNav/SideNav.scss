
@import '../../assets/breakpointManager.scss';

.side-nav-component {
	position: absolute;
	display: flex;
	flex-direction: column;
	justify-content: space-between;
	align-items: center;
	border: 2px #2c3e50 solid;
	height: auto;
	width: 35px;
	border-radius: 80px;
	z-index: 20;
	top: 30%;
	cursor: pointer;
	transition: all 0.4s ease-out;
	transform: translate(0px);
  
	.label {
		opacity: 0;
		position: absolute;
		width: auto;
		height: 20px;
		transition: all 0.3s ease-out;
		border-radius: 5px;
    padding: 5px;
    font-size: 10px;
    white-space: nowrap;
	}

	.shadow {
		border: 10px solid white;
		&:hover {
			.label {
			  opacity: 45;
				transform: translate(25px, -5px);
			}
			border: 10px solid #c3cad1;
		}
	}

	&__circle {
		&:hover {
			background-color: white;
		}

		height: 15px;
		width: 15px;
		background-color: #2c3e50;
		border-radius: 50%;
  }
  
  &--hide {
    transform: translate(-150px);
  }

  @include for-side-nav {
    transform: translate(-150px);
  }   
}