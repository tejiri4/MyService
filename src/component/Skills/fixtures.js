export const skillsArray = [
  {
    skill: 'Photography',
    percentage: '90%',
    color: '#786DBB'
  },
  {
    skill: 'Web design',
    percentage: '70%',
    color: '#38DACF'
  },
  {
    skill: 'Web development',
    percentage: '90%',
    color: '#3088C7'
  },
  {
    skill: 'Mobile application',
    percentage: '80%',
    color: '#DE3AAF'
  }
];
