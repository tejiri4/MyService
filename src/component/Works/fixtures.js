export const works = [
  {
    name: "ALL",
    key: "all",
    data: [
      'https://res.cloudinary.com/store-manager/image/upload/v1562948655/balazs-ketyi-6ba_vdgx_go-unsplash.jpg',
      'https://res.cloudinary.com/store-manager/image/upload/v1562948760/tracy-adams-rA5MU4pXnWE-unsplash.jpg',
      'https://res.cloudinary.com/store-manager/image/upload/v1562948603/balazs-ketyi-LPWl2pEVGKc-unsplash.jpg',
      'https://res.cloudinary.com/store-manager/image/upload/v1562948655/balazs-ketyi-6ba_vdgx_go-unsplash.jpg',
      'https://res.cloudinary.com/store-manager/image/upload/v1562948760/tracy-adams-rA5MU4pXnWE-unsplash.jpg',
      'https://res.cloudinary.com/store-manager/image/upload/v1562949962/ben-kolde-Ajcipx1VDXI-unsplash.jpg',
    ]
  },
  {
    name: "WEBDESIGN",
    key: "webdesign",
    data: [
      'https://res.cloudinary.com/store-manager/image/upload/v1562948655/balazs-ketyi-6ba_vdgx_go-unsplash.jpg',
      'https://res.cloudinary.com/store-manager/image/upload/v1562948760/tracy-adams-rA5MU4pXnWE-unsplash.jpg',
      'https://res.cloudinary.com/store-manager/image/upload/v1562948603/balazs-ketyi-LPWl2pEVGKc-unsplash.jpg',
    ]
  },
  {
    name: "WORDPRESS",
    key: "wordpress",
    data: [
      'https://res.cloudinary.com/store-manager/image/upload/v1562948655/balazs-ketyi-6ba_vdgx_go-unsplash.jpg',
      'https://res.cloudinary.com/store-manager/image/upload/v1562948760/tracy-adams-rA5MU4pXnWE-unsplash.jpg',
      'https://res.cloudinary.com/store-manager/image/upload/v1562948603/balazs-ketyi-LPWl2pEVGKc-unsplash.jpg',
    ]
  },
  {
    name: "BRANDING",
    key: "branding",
    data: [
      'https://res.cloudinary.com/store-manager/image/upload/v1562948655/balazs-ketyi-6ba_vdgx_go-unsplash.jpg',
      'https://res.cloudinary.com/store-manager/image/upload/v1562948760/tracy-adams-rA5MU4pXnWE-unsplash.jpg',
      'https://res.cloudinary.com/store-manager/image/upload/v1562948603/balazs-ketyi-LPWl2pEVGKc-unsplash.jpg',
    ]
  }
];
