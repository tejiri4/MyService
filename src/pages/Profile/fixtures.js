export const navArray = [
  {
    name: "HOME",
    key: "home"
  },
  {
    name: "ABOUT ME",
    key: "about-me"
  },
  {
    name: "SERVICE",
    key: "service"
  },
  {
    name: "PORTFOLIO",
    key: "portfolio"
  },
  {
    name: "BLOG",
    key: "blog"
  },
  {
    name: "CONTACT",
    key: "contact"
  }
];
