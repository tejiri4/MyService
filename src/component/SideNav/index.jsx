// react libraries
import React from 'react';
import PropTypes from 'prop-types';

// styles
import './SideNav.scss';

// fixtures
import { navArray } from '../../pages/Profile/fixtures';

const SideNav = props => {
  const { classes } = props;

  const renderSideNavDetails = navDetail => (
    <div className='side-nav-component__circle shadow' key={navDetail.key} >
      <div className='label'>{navDetail.name}</div>
    </div>
  );

  return (
    <div className={`side-nav-component${classes}`}>
      {
        navArray.map(navDetail => renderSideNavDetails(navDetail))
      }
    </div>
  );
};

SideNav.propTypes = {
  classes: PropTypes.string
};

export default SideNav;
