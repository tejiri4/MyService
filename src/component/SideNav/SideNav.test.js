import React from 'react';
import { shallow } from 'enzyme';
import SideNav from './';
import { navArray } from '../../pages/Profile/fixtures';

describe('NavBar component', () => {
  let wrapper;

  beforeEach(() => {
    const props = {
      classes: ''
    };
    wrapper = shallow(<SideNav {...props} />);
  });

  it('renders the SideNav component without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});