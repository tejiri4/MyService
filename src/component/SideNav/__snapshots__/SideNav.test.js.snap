// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar component renders the SideNav component without crashing 1`] = `ShallowWrapper {}`;
